AtReconstruction/AtFilter/AtFilterZero.cxx

```diff
@@ -1,8 +1,9 @@
 #include "AtFilterZero.h"
 
 #include "AtPad.h"
+#include "AtPadReference.h"
```

Comment:
Basically replace all of the includes with forward deceleration unless we are actually access the data in the pointer.